### DISCUSSION TITLE
Create and Connect multiple Rows

### DISCUSSION THREAD

**Discussion by pasekit** [prisma/prisma#3559] [Q&A] [answered]
Hi, 

i have the following problem.
I create a new row in the database with `this.prisma.user.create({...})`
Then i want to connect all posts with ids less than 10 for example.

My problem:
if i use "update" i can use  
`this.prisma.post.update({ where: ..., data: { user: { connect: { id: ouputFromCreate.id } } } })`
if i use "updateMany" i can use
`this.prisma.post.updateMany({ where: {id: { lt: 10} }, data: ... })`

I think i need both together but that isn't allow. 
Need somethink like:
output = `INSERT INTO user SET name = 'data.name'`
`UPDATE post SET userId = 'output.id' WHERE id < 10`

where did I make my mistake?
Can I summarize this in one SQL command?

`

**Comment by ryands17** [comment, marked as answer]
@pasekit 👋 
You can do it like this:

```ts
let postIds = []
  for (let i = 1; i < 10; i++) {
    postIds.push({ id: i })
  }
  await prisma.user.create({
    data: {
      name: 'user 1',
      posts: {
        connect: postIds,
      },
    },
  })
```

**Reply by pasekit** [reply]
not a very elegant solution, but it works.

Thank you

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
